homeassistant/components/shopping_list/services.yaml

```diff
@@ -10,3 +10,13 @@ complete_item:
     name:
       description: The name of the item to mark as completed.
       example: Beer
+restore_item:
+  description: Marks an item as incomplete in the shopping list.
+  fields:
+    name:
+      description: The name of the item to mark as incomplete.
+      example: Beer
+complete_list:
+  description: Marks all items as completed in the shopping list. It does not remove the items.
+restore_list:
```

Comment:
restore feels a bit weird as a name for this? As it might sound like it is restoring a state, but in reality, it is unchecking all boxes. uncomplete_all?